client/charts/js/components/FilterSummary.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import classNames from 'classnames'
+
+const symbolMap = {
+	"Arrest / Criminal Charge": "arrest",
+	"Border Stop": "border_stop",
+	"Denial of Access": "denial_of_access",
+	"Equipment Search or Seizure": "equipment_search",
+	"Assault": "assault",
+	"Leak Case": "leak_case",
+	"Subpoena / Legal Order": "subpoena",
+	"Equipment Damage": "equipment_damage",
+	"Prior Restraint": "prior_restraint",
+	"Chilling Statement": "chilling_statement",
+	"Other Incident": "other_incident",
+}
+
+export default function FilterSidebar() {
+	const searchParams = new URLSearchParams(window.location.search)
+	const {
+		categories: categoriesStr,
+		tags: tagsStr,
+		date_lower,
+		date_upper,
+		...restFilters
+	} = Object.fromEntries(searchParams)
+
+	const categories = categoriesStr ? categoriesStr.split(',').map(d => d.trim()) : []
+	const tags = tagsStr ? tagsStr.split(',').map(d => d.trim()) : []
+
+	const clearFilter = (filterKey, newFilterValue) => {
+		const url = new URL(window.location);
+		const newSearchParams = new URLSearchParams(window.location.search)
+
+		if (newFilterValue) newSearchParams.set(filterKey, newFilterValue)
+		else newSearchParams.delete(filterKey)
+
+		url.search = newSearchParams.toString()
+		window.location = url.toString()
+	}
+
+	return (
+		<ul className="filters-summary">
+			{categories.map(category => (
+				<li key={category}>
+					<button
+						onClick={() => clearFilter(
+							"categories", categories.filter(d => d !== category).join(',')
+						)}
+						className="btn btn-tag"
+						tabIndex={0}
```

Comment:
There is no need to add `tabIndex=0` on a button I think since it is already interactive.